Javascript/QuizApp/display.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz Application Project</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <button id="theme-toggle" class="toggle-button">Change Theme</button>
+
+      <div class="heading-container">
+        <div class="logo-container">
+          <img src="logo/logo.png" alt="Logo" class="logo" />
+        </div>
+
+        <div class="heading-nav-container">
+          <h1>Quiz Application Project</h1>
+          <h2>Welcome to All Quiz Questions Display PAGE</h2>
+          <nav>
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="inputform.html">Input</a></li>
+              <li><a id="search" href="search.html">Search</a></li>
+              <li><a id="display" href="#">Display</a></li>
+              <li><a id="game" href="game.html">Competition</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main>
+      <div class="message-container">
+        <p class="message-text"></p>
+      </div>
+      <div class="quizList-container-display"></div>
+    </main>
+
+    <footer>
+      <footer>
+        <p>&copy; 2025 HYF. All Rights Reserved.</p>
+        <p>
+          Contact: <a href="tel:+4553819612">Phone: +45 53819612 </a> |
+          <a href="wasimhussain3@gmail.com">Email: wasimhussain3@gmail.com</a
+          ><br /><a
+            href="https://www.linkedin.com/in/sayyed-wasim-hussain-bukhari-041342299/"
+            >LinkedIn</a
+          >
+          |
+          <a href="https://github.com/Bukhari1214">GitHub</a>
+        </p>
+      </footer>
+    <script src="commonFunction.js"></script>
+    <script src="fetchData.js"></script>
+    <script src="diaplayQuiz.js"></script>
```

Comment:
```suggestion
    <script src="displayQuiz.js"></script>
```
The file name needs to change too.